externs/browser/url.js

```diff
@@ -22,10 +22,20 @@
  * @author rdcronin@google.com (Devlin Cronin)
  */
 
+/**
+ * @typedef {IArrayLike<string>}
+ */
+var URLSearchParamsTupleType;
+
+/**
+ * @typedef {IArrayLike<!URLSearchParamsTupleType>|Iterable<!URLSearchParamsTupleType>}
+ */
+var URLSearchParamsTupleSequenceType;
```

Comment:
Per my reasoning in the comment above, I think we should just drop this definition.

The `@param` below then becomes.
`{(string|!Iterable<!URLSearchParamsTupleType>|!Object<string,string>)=} init`